### PR TITLE
Cleanup uploaded files for PSR-15 handlers

### PR DESCRIPTION
**Pull request by mnapoli**

### CODE REVIEW

File: src/Event/Http/Psr7Bridge.php [hunk 1]
```diff
@@ -166,4 +168,19 @@ private static function parseKeyAndInsertValueInArray(array &$array, string $key
...
+    /**
+     * Cleanup previously uploaded files.
+     */
+    public static function cleanupUploadedFiles(): void
+    {
+        $tmpFiles = glob(sys_get_temp_dir() . '/' . self::UPLOADED_FILES_PREFIX . '*');
+        if ($tmpFiles !== false) {
+            foreach ($tmpFiles as $file) {
+                if (is_file($file)) {
+                    unlink($file);
```
Comment (GrahamCampbell) [inline src/Event/Http/Psr7Bridge.php:181]: This could fail if the previous request didn't cleanup the open handles properly.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Event/Http/Psr7Bridge.php (hunk 1 above)
Comment (mnapoli) [reply src/Event/Http/Psr7Bridge.php:181]: In which case is there anything we can do? I don't want to turn this into an error and break stuff

Comment (GrahamCampbell) [reply src/Event/Http/Psr7Bridge.php:183]: Maybe we need to silence it?

Comment (mnapoli) [reply src/Event/Http/Psr7Bridge.php]: 🤦 I forgot about custom error handlers that might turn the warning into exceptions! (I was focused on the `bool` return). The joys of warnings.

**Review by GrahamCampbell — APPROVED** [review]

**Pull request merged**